app/views/layout.nunjucks

```diff
@@ -20,7 +20,7 @@
     <meta name="DCSext.ServiceName" content="GP Appointment Finder" />
     {% block meta %}{% endblock %}
 
-    {% set qs = '?v=5' %} <!-- DO NOT USE 6 -->
+    {% set qs = '?v=6' %} <!-- DO NOT USE 6 -->
```

Comment:
Is there a reason the comment that says `DO NOT USE 6` is being ignored? Please change it to 7. You might as well delete the comment at the time.